Taylor_series_Repo/Taylor_Series_Horner_Rule.cpp

```diff
@@ -0,0 +1,29 @@
+// Taylor series using Horner Rule in c++
+
+#include<bits/stdc++.h>
+using namespace std;
+
+// creating exponential function
+double exponential (int x, int num)
+{
+    static double s; // declare static variable
+    if(num == 0)   // Base case
+    {
+        return s;
+    }
+    s = 1 + x * (s / num); // formula of horner's rule
+    return exponential (x, num - 1); // recursive case
+}
+// main method
+int main ()
+{
+    int x, num;
+    cin >> x >> num;

```

Comment:
Space around ``>>`` & ``<<`` is not necessary in ``cin`` & ``cout`` statements.

Reply:
`>>` and `<<` are operators. There should be space around them.

Reply:
@marwahmanbir, @SakshiShreya, What should I do?